cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -1664,6 +1666,34 @@ impl PrettyPrint for Inst {
                 format!("j{} ; ud2 {} ;", cc.invert().to_string(), trap_code)
             }
 
+            Inst::TrapIfAnd {
+                cc1,
+                cc2,
+                trap_code,
+                ..
+            } => {
+                format!(
+                    "j{} ; j{} ; ud2 {} ;",
+                    cc1.invert().to_string(),
+                    cc2.invert().to_string(),
+                    trap_code
+                )
+            }
+
+            Inst::TrapIfOr {
+                cc1,
+                cc2,
+                trap_code,
+                ..
+            } => {
+                format!(
+                    "j{} ; j{} ; ud2 {} ;",
```

Comment:
I'm not sold on this pretty-printing format. Do you have ideas on how better to convey what's going on? Maybe something like `j({} || {})`?

Reply:
(I'll look at the rest in the detail tomorrow but one thought here: )

This is actually kind of a fun historical oddity: when initially implementing the new backends we kept the pretty-printing format to something that could actually feed into GNU `as`. The first "hello world" (fib function actually) on the aarch64 backend was a thing I captured from the trace logs and then assembled and linked manually. Since then we've sort of drifted from that and the pretty-printing just has to be "comprehensible to a deeply tired debugging human"; so something that clarifies here is welcome :-)

I might even go so far as `trap_if_or CC, CC` or somesuch...